stories/dynamic-page/dynamic-page.stories.js

```diff
@@ -0,0 +1,954 @@
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+            </div>
+        </div>
+        <div class="fd-bar fd-bar--floating-footer" style="position:relative">
+            <div class="fd-bar__right">
+                <div class="fd-bar__element">
+                    <button aria-label="button" class="fd-button fd-button--emphasized fd-button--compact">Save</button>
+                </div>
+                <div class="fd-bar__element">
+                    <button aria-label="button" class="fd-button fd-button--transparent fd-button--compact">Cancel</button>
+                </div>
+            </div>
+        </div>
+    </div>
+    <br /><br />
+`;
+
+desktop.storyName = 'Desktop';
+desktop.parameters = {
+    docs: {
+        iframeHeight: 150,
+        storyDescription: `The dynamic page displays on desktop screens with the L, XL, and XXL sizes. To optimize dynamic page for desktop, add these modifier classes to the following elements:
+        
+| Element | Modifier class |
+| ----------------: | :------------ |
+| Title Area | \`fd-dynamic-page__title-area--xl\` |
+| Header | \`fd-dynamic-page__header--xl\` |
+| Tabs | \`fd-dynamic-page__tabs--xl\` |
+| Content | \`fd-dynamic-page__content--xl\` |
+
+Note: The XL size is presented in this table, however the L and XXL size works as well.
```

Comment:
I think it would be useful to list all classes we support.

Reply:
added two new sections at the top called `Modifiers` and `Background variants` covering the supported modifiers which were not explicitly mentioned. The ones with the sizes `--sm` etc are not added here as they are already present in each example type. Hope this is okay.
![image](https://user-images.githubusercontent.com/53509521/94121954-93a35000-fe6f-11ea-9324-e294aceab90c.png)

@jessmetivier  please take a look as there has been an addition to the docs.